docs/pages/guides/migrating-to-react-native-firebase.mdx

```diff
@@ -0,0 +1,165 @@
+]} />
+
+If you are using [Expo Prebuild](/guides/adopting-prebuild) (also known as Managed Workflow) and building your app for iOS, install the `expo-build-properties` plugin and apply the following configuration to your [Expo config](/workflow/configuration/) file.
+
+<Terminal cmd={["$ npx expo install expo-build-properties"]} />
+
+### Add Config plugin to Expo config file
+
+In the Expo config file, add add `expo-community-flipper` to the `plugins` array with the following configuration:
+
+```json app.json
+{
+  "expo": {
+    "plugins": [
+      [
+        "expo-build-properties",
+        {
+          "ios": {
+            "useFrameworks": "static"
+          }
+        }
+      ]
+    ]
+  }
+}
+```
+
+If you are using the Bare Workflow, see [React Native Firebase's installation instructions](https://rnfirebase.io/#bare-workflow).
+
+## Update analytics requests
+
+Firebase Analytics requests in React Native Firebase are almost the same - mostly, they are imported a little differently. Update your code as shown here:
+
+```diff
+// update imports
+- import * as Analytics from "expo-firebase-analytics";
++ import analytics from "@react-native-firebase/analytics";
+
+// update each request to analytics
+- Analytics.logEvent("test_analytics_event", {
+-  additionaParam: "test",
+- });
++ analytics().logEvent("test_analytics_event", {
++  additionaParam: "test",
++ });
+```
+
+`logEvent` is asynchronous, so be sure to use `await` in front of the function call if you are using it in an `async` function.
+
+## Creating a development build
+
+If you have previously used Expo Go for development and testing, at this point, you will need to migrate to a [development build](/development/introduction/#from-expo-go-to-development-builds), as Expo Go does not include the native code used in React Native Firebase.
+
+[Follow these instructions for creating a development build](/development/getting-started/). Once you are finished, you should have an **eas.json** file that includes a build profile that looks like this:
+
+```json eas.json
+{
+  "build": {
+    "development": {
+      "developmentClient": true,
+      "distribution": "internal"
+    }
+  }
+}
+```
+If you wish to run the [development build on an iOS simulator](/build-reference/simulators/) instead of a device, include the `ios.simulator` in your profile:
+
+```json eas.json
+{
+  "development": {
+      //..
+      "ios": {
+        "simulator": true
+      }
+    },
+}
+```
+`expo-dev-client` should be present in your **package.json**. Once you run [`eas build`](/build/#running-a-build) with your development build [profile](/build/eas-json/#build-profiles) and platform of choice, you will have an app binary file that you can install on your device or simulator.
+
+If you are already using a development build, rebuild your app, as your native code dependencies have changed.
+
+## Test your changes
+
+Test your changes locally on your Development Build by running:
```

Comment:
```suggestion
Test your changes locally on your development build by running:
```

in this doc both "Development Build" and "development build" are used - should say "development build"